manifests/c/Caphyon/AdvancedInstaller/20.2/Caphyon.AdvancedInstaller.locale.en-US.yaml

```
@@ -0,0 +1,15 @@
+# Automatically updated by the winget bot at 2022/Dec/14
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Caphyon.AdvancedInstaller
+PackageVersion: "20.2"
+PackageLocale: en-US
+Publisher: Caphyon
+PackageName: Advanced Installer
+PackageUrl: https://AdvancedInstaller.com/
+License: © Caphyon 2002 - 2020 Ltd. All rights reserved.

```

```suggestion
License: Proprietary
```
License needs fixing